test/fixtures/active_record.rb

```diff
@@ -89,7 +89,7 @@ class ObjectTag < ActiveRecord::Base
 end
 class PolymorphicObjectTagSerializer < ActiveModel::Serializer
   attributes :id
-  has_many :taggable, serializer: PolymorphicSimpleSerializer, polymorphic: true
+  belongs_to :taggable, serializer: PolymorphicSimpleSerializer, polymorphic: true
```

Comment:
Not sure why the belongs_to relationships was specified with a has_many serializer method here and in the PolymorphicBelongsToSerializer below.  But this change surfaces the bug reported in #2160